package.json

```diff
@@ -33,7 +33,8 @@
     "services:clean": "node bin/services-clean.js",
     "prepare": "husky install",
     "preinstall": "npx only-allow pnpm",
-    "postinstall": "cd src/mobile && npm install"
+    "postinstall": "cd src/mobile && npm install",
+    "mobile": "cd src/mobile && expo start"
```

Comment:
I don't think we should add scripts to the root.  Let's normalize going into the sub-project and doing things.

Reply:
so what you mean is that we still let the users to `cd src/mobile` and run `pnpm script` inside the `mobile` directory ?